### DISCUSSION TITLE
Create Index with Authentication Disabled

### DISCUSSION THREAD

**Discussion by ghost** [appy-one/acebase-server#24] [Q&A] [answered]
I am using AceBase Server as a private database in a docker environment with Node-Red. I have authentication turned off since I implement security in Node-Red and figured it would reduce overhead on database requests. However, when I attempt to create an index I get an error that it can only be performed by an 'Admin'.

Is this a defect, or by design? Is there a way to connect as 'Admin' with the AceBase client with authentication disabled.

**Comment by appy-one** [comment, marked as answer]
Sorry, I assumed that by creating an issue from this discussion it would be linked to it. Apparently not, see #26. I'll be performing more tests today, a new release that fixes this is on the horizon!

**Reply by ghost** [reply]
Hey, I appreciate the heads up. I will keep my eye on the issue and upcoming releases.
